src/config/env.web.js

```diff
@@ -0,0 +1 @@
+export default process.env
```

Comment:
is this 100% working with webpack ? 
as i understood define plugin is looking for the whole strings "process.env.REACT_APP_xxxxx"

please recheck this on the web.

Reply:
I tested and it's working well